pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -325,6 +325,9 @@ public void setTextIndexColumns(Set<String> textIndexColumns) {
     _textIndexColumns = textIndexColumns;
   }
 
+  @VisibleForTesting
+  public void setJsonIndexColumns(Set<String> jsonIndexColumns) { _jsonIndexColumns = jsonIndexColumns; }
```

Comment:
Reformat into 3 lines, and put it after the `setFSTIndexColumns` because text and fst index are both for text search

Reply:
done